man/kernel-install.xml

```diff
@@ -161,8 +161,12 @@
 
   <refsect1>
     <title>Environment variables</title>
+
     <para>If <option>--verbose</option> is used, <varname>$KERNEL_INSTALL_VERBOSE=1</varname> will be set for
     the plugins. They may output additional logs in this case.</para>
+
+    <para>If <varname>MACHINE_ID=</varname> is set and not empty, it will be used as <replaceable>MACHINE-ID</replaceable>,
+    overriding any automatic detection attempts, and must be 32 hexadecimal characters.</para>
```

Comment:
", and must be 32 hexadecimal charcters" → ". The value must be a valid machine ID (32 hexadecimal characters)" or something like that.